pages/account/customer/product.de-de.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Account informations
+title: Account Informationen
```

Comment:
Hi! Thanks :)
Can you change "Account Informationen" for "Accountinformationen" please?